WordPress/Classes/Utility/LinkBehavior.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import WebKit
+
+enum LinkBehavior {
+    case all
+    case hostOnly(URL)
+    case urlOnly(URL)
+
+    func handle(navigationAction: WKNavigationAction, for webView: WKWebView) -> WKNavigationActionPolicy {
+
+        // We only want to apply this policy for links, not for all resource loads
+        guard navigationAction.navigationType == .linkActivated else { return .allow }
```

Comment:
Per our [style guide](https://github.com/wordpress-mobile/swift-style-guide), `Closing braces should always be placed on a new line regardless of the number of enclosed statements.`